### PR TITLE
add features to print "Hello, world" message on port 3000

### PR DESCRIPTION
**Pull request by ajstyles0524**
What does this change do?
Added feature to print Hello, world on port 3000 in Rust code

Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Are there any changes pending?
No

Does any team have to be notified of changes in this feature?
Yes

Definition of Done:
- [x] Is there >90% unit test code coverage?
- [x] Does this PR add new dependencies? If so, please list out the same.
      rocket = "0.5.0-rc.1"
      log = "0.4.14"
- [ ]  Will this feature require a new piece of infrastructure to be implemented?
- [x] Is there appropriate logging included?
- [x] Does the project compile ok?
- [x] Have Clippy violations been fixed?
- [x] Have Code is properly formatted?

### CODE REVIEW

**Review by Sandeep-Knoldus — APPROVED** [review]

**Review by Rohit0823 — APPROVED** [review]

File: Assignments/rocket_web/src/launch.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+use log::debug;
+#[get("/")]
+/// world is a handler which is used to process the request and return a response
```
Comment (bishtpawan) [inline Assignments/rocket_web/src/launch.rs:3]: please update documentation

File: Assignments/rocket_web/src/launch.rs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+///
+/// #Return
+///
+/// Return &'static str
+fn hello() -> &'static str {
+    debug!("{}", "some log message");
+    "Hello, world!"
+}
+#[launch]
+/// rocket is a function which is used to startq the server
```
Comment (bishtpawan) [inline Assignments/rocket_web/src/launch.rs:17]: please update documentation

File: Assignments/rocket_web/src/launch.rs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+///
+/// #Arguments
+///
+/// No Arguments
+///
+/// #Return
+///
+/// Return &'static str
+fn hello() -> &'static str {
+    debug!("{}", "some log message");
```
Comment (bishtpawan) [inline Assignments/rocket_web/src/launch.rs:13]: please provide proper log messages

File: Assignments/rocket_web/src/launch.rs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+/// rocket is a function which is used to startq the server
+///
+/// #Arguments
+///
+/// No Arguments
+///
+/// #Return
+///
+/// Return Rocket<Build>
+pub fn launch() -> _ {
```
Comment (bishtpawan) [inline Assignments/rocket_web/src/launch.rs:26]: why we have _ in the return type?

File: Assignments/rocket_web/src/test.rs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#[cfg(test)]
+mod test {
```
Comment (bishtpawan) [inline Assignments/rocket_web/src/test.rs:2]: please add negative test case as well

**Review by bishtpawan — CHANGES_REQUESTED** [review]
Please update naming conventions, incorporate error handling, use proper documentation.

**Pull request closed without merging**